vsphere/datadog_checks/vsphere/vsphere.py

```diff
@@ -340,7 +340,12 @@ def refresh_infrastructure_cache(self):
             mor_payload["tags"] = tags  # type: Dict[str, Any]
 
             if hostname:
-                mor_payload['hostname'] = hostname
+                if self._config.hostname_to_upper:
+                    mor_payload['hostname'] = hostname.upper()
+                elif self._config.hostname_to_lower:
+                    mor_payload['hostname'] = hostname.lower()
+                else:
+                    mor_payload['hostname'] = hostname
```

Comment:
```suggestion
                if self._config.hostname_to_upper:
                    hostname = hostname.upper()
                elif self._config.hostname_to_lower:
                    hostname = hostname.lower()

               mor_payload['hostname'] = hostname
```
what do you think about something like this?

Reply:
I basically did this, but with `hostname_transform`.